tests/analyzer/sourcecode/bidirectional-characters.py

```diff
@@ -0,0 +1,11 @@
+def demo():
```

Comment:
curious if we have some real-world samples using this techniques? if yes, it would be great to copy/paste their actual code in there

Reply:
Unfortunately I wasn't able to find no malware implementing this attack